src/main/java/Duke.java

```diff
@@ -1,10 +1,68 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+    static final String LOGO = "$$$$$$$\\                  $$\\           \n" +
+            "$$  __$$\\                 $$ |          \n" +
+            "$$ |  $$ |$$\\   $$\\  $$$$$$$ | $$$$$$\\  \n" +
+            "$$ |  $$ |$$ |  $$ |$$  __$$ |$$  __$$\\ \n" +
+            "$$ |  $$ |$$ |  $$ |$$ /  $$ |$$$$$$$$ |\n" +
+            "$$ |  $$ |$$ |  $$ |$$ |  $$ |$$   ____|\n" +
+            "$$$$$$$  |\\$$$$$$  |\\$$$$$$$ |\\$$$$$$$\\ \n" +
+            "\\_______/  \\______/  \\_______| \\_______|";
+    static final String GREET = "I am Dude.\n" +
+            "Type something in, dude.";
+    static final String BYE = "Catch you later, dude.";
```

Comment:
LOGO is okay but GREET and BYE in your context are verbs and they are a constant variable so they should be named like nouns. For eg, GREETING and GOODBYE